GuessingGame/Service.cs

```diff
@@ -37,35 +37,35 @@ public int GetNumber()
         Console.Write(" Choose level (Easy/Medium/Hard) : ");
         string difficulty = Console.ReadLine();
         
-                    switch (difficulty.ToLower())
-                    {
-                    case "easy":
-                    return GetResult(1,5,5);
-                    case "medium":
-                    return GetResult(1,10,10);  
-                    case "hard":
-                    return GetResult(1,15,15);
-                    default:
-                    Console.WriteLine("enter a correct string");
-                        return GetLevel();
-                       
-                    }
+            switch (difficulty.ToLower())
+            {
+            case "easy":
+            return GetResult(1,5,5);
+            case "medium":
+            return GetResult(1,10,10);  
+            case "hard":
+            return GetResult(1,15,15);
+            default:
+            Console.WriteLine("enter a correct string");
+                return GetLevel();
+                
+            }
 
     }
     public string GetResult(int minRange,int maxRange,int multiplier){
-                    Random random = new Random();
-                    int randomNumber = random.Next(minRange, maxRange);
+        Random random = new Random();
```

Comment:
you have dependency injection in this class but you aren't implementing it here.